### PR TITLE
Install MicroBuild code signing plugin and set SignType = real

### PR DESCRIPTION
**Pull request by natemcmaster**
Part of https://github.com/aspnet/Coherence-Signed/issues/694

This makes available the infrastructure to run code signing. Additional work is necessary to plumb aspnet bits into the signing tools.

### CODE REVIEW

**Comment by natemcmaster** [comment]
@HaoK can you take a look?

File: .vsts/builds/ci-official.yml [hunk 1]
```diff
@@ -18,11 +18,19 @@ phases:
   variables:
     CI: true
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+    TeamName: AspNetCore
+    _SignType: real
```
Comment (HaoK) [inline .vsts/builds/ci-official.yml]: is this just to signify its a private var used in this yml only?  just curious what the pattern is

Comment (HaoK) [reply .vsts/builds/ci-official.yml]: Also what are the other sign types, test? (maybe worth a comment for posterity)

Comment (natemcmaster) [reply .vsts/builds/ci-official.yml]: SignType = { test, real }

I actually used the underscore for a different reason. VSTS variables automatically become environment variables, and this one was causing issues in MSBuild projects which use the `$(SignType)` msbuild prop.

Comment (HaoK) [reply .vsts/builds/ci-official.yml]: Ok cool, good to know :)

Comment (natemcmaster) [reply .vsts/builds/ci-official.yml]: I added this to the yml as comments for future reference.

**Pull request merged**